src/components/Balloon/index.js

```diff
@@ -27,14 +27,14 @@ function Balloon({ startAfter = 3000, duration = 15000 }) {
           <div className="balloon__header">
             <img className="balloon__header__img" src={risk} alt="risk" />
             <span className="balloon__header__text">
-              Your computer might be at risk
+              Cookie agreement
             </span>
           </div>
           <p className="balloon__text__first">
-            Antivirus software might not be installed
+            This website uses cookies to store site data like minesweeper highscores. 
           </p>
           <p className="balloon__text__second">
-            Click this balloon to fix this problem.
+            Click this balloon to accept the cookies. 
```

Comment:
This seems a bit conflicting to the user... it doesn't actually accept the cookies. This *could* be worse then not telling the user about the cookies, you are implying that they ***can*** opt out of the cookies.  

Another note is that, at least where GDPR applies, you must explain the data that is being collected saying words such as "like" do not make up a privacy policy. 

Here is an example of something that could be said: 
> This site uses cookies to store data (including, but not limited to, Minesweeper highscores). If you do not agree to the storing of such data do not use this site, or the features that collect such data.